### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by hbons**

### CODE REVIEW

File: FigmaSharp.Controls/FigmaSharp.Controls.Cocoa/Converters/CocoaConverter.cs [hunk 1]
```diff
@@ -95,27 +95,27 @@ public override string ConvertToCode(CodeNode currentNode, CodeNode parentNode,
 
 					if (CanSetAccessibilityRole && currentNode.Node.IsA11Group()) {
 						var fullRoleName = $"{typeof(AppKit.NSAccessibilityRoles).FullName}.{nameof(AppKit.NSAccessibilityRoles.GroupRole)}";
-						builder.WritePropertyEquality(currentNode.Name, nameof(AppKit.NSView.AccessibilityRole), fullRoleName);
```
Comment (netonjm) [inline FigmaSharp.Controls/FigmaSharp.Controls.Cocoa/Converters/CocoaConverter.cs:98]: why this change?

Comment (hbons) [reply FigmaSharp.Controls/FigmaSharp.Controls.Cocoa/Converters/CocoaConverter.cs:98]: we don't need this feature for now. we only need it in very specific cases and we can add them in the code-behind. I want to improve this in another PR.

File: FigmaSharp/FigmaSharp/Services/Nodes/CodeNode.cs [hunk 2]
```diff
@@ -40,6 +40,7 @@ public CodeNode (FigmaNode node, string name = null, bool isClass = false, CodeN
 		public CodeNode Parent { get; private set; }
 
 		public string Name { get; set; }
+		public string OriginalName => Node.name;
```
Comment (netonjm) [inline FigmaSharp/FigmaSharp/Services/Nodes/CodeNode.cs:43]: this is not used

File: FigmaSharp/FigmaSharp.Cocoa/Converters/StackViewConverter.cs [hunk 3]
```diff
@@ -100,6 +103,9 @@ public void ConfigureCodeProperty(string propertyName, CodeNode codeNode, String
         {
             var frame = (FigmaFrame)codeNode.Node;
 
+            if (!codeNode.Node.visible)
+                code.WritePropertyEquality(codeNode.Name, nameof(NSView.Hidden), true);
```
Comment (netonjm) [inline FigmaSharp/FigmaSharp.Cocoa/Converters/StackViewConverter.cs:107]: why this only happens in StackViewConverter ?

File: FigmaSharp/FigmaSharp/Services/Providers/NodeProvider.cs [hunk 4]
```diff
@@ -78,6 +78,9 @@ public void Load (string file)
 			}
 		}
 
+		public static readonly string SkipCharacters = "//";
+		public static readonly string SkipKeyword = "!skip";
```
Comment (netonjm) [inline FigmaSharp/FigmaSharp/Services/Providers/NodeProvider.cs:82]: this new static methods are not used

File: FigmaSharp.Controls/FigmaSharp.Controls.Cocoa/Converters/CocoaConverter.cs (hunk 1 above)
Comment (netonjm) [reply FigmaSharp.Controls/FigmaSharp.Controls.Cocoa/Converters/CocoaConverter.cs:98]: I don't like remove features without a code replacement, what is the reason don't change it from the document?

File: FigmaSharp/FigmaSharp.Cocoa/Converters/StackViewConverter.cs (hunk 3 above)
Comment (hbons) [reply FigmaSharp/FigmaSharp.Cocoa/Converters/StackViewConverter.cs:107]: this logic is copied from the base CocoaConverter. StackViewConverter doesn't have it as a parent. I think this class should be moved to .Controls later.

File: FigmaSharp/FigmaSharp/Services/Nodes/CodeNode.cs (hunk 2 above)
Comment (hbons) [reply FigmaSharp/FigmaSharp/Services/Nodes/CodeNode.cs:43]: I will create a new PR that won't have the skip commits.

File: FigmaSharp.Controls/FigmaSharp.Controls.Cocoa/Converters/CocoaConverter.cs (hunk 1 above)
Comment (hbons) [reply FigmaSharp.Controls/FigmaSharp.Controls.Cocoa/Converters/CocoaConverter.cs:98]: You're right. I think what we should do here is check if the a11y layer names in the instance have changed compared to the main component and we should only generate these properties if they've been changed by the user.

**Review by netonjm — CHANGES_REQUESTED** [review]
This PR must be divided into atomic parts to facilitate the review and include a descriptive message (or link a problem associated with it), this is done so that in the event of a regression it facilitates an understanding of why the change was made and make it easier to reverse it. In case of small related changes I'm ok to merge it.
https://github.com/microsoft/FigmaSharp/blob/main/CONTRIBUTING.md
